src/runners/http_staking_service_runner.ts

```diff
@@ -1,59 +0,0 @@
-/**
```

Comment:
Is staking finally getting its own service? 🙏  Let's make sure that the website is correctly hooked up before we ship this 😄